src/javascript/expression.rs

```diff
@@ -326,6 +334,34 @@ impl<'module> Generator<'module> {
             .append(" ")
             .append(right))
     }
+
+    fn todo<'a>(&mut self, message: &'a Option<String>, location: &'a SrcSpan) -> Document<'a> {
+        self.tail_position = false;
```

Comment:
I set tail position false here because we are permanently not in the tail position as from here the function will not return.
i.e. we don't want to render
```
return throw error
```
just
```
throw error
```